### DISCUSSION TITLE
Crash while sharing a repository

### DISCUSSION THREAD

**Discussion by abhinavsatheesh** [community/community#56675] [Mobile]
While sharing a repository in GitHub for ios, the app crashes
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.114.0 (99800193)
Device: iPad Pro 9.7-inch (iOS 15.6.1)
</details>

**Comment by alcere** [comment]
@abhinavsatheesh thanks for reporting! I've created an issue and we will investigate the problem.
